sql/moz-fx-data-shared-prod/telemetry_derived/desktop_funnel_activation_day_6_v1/query.sql

```diff
@@ -65,15 +68,17 @@ SELECT
   attribution_source,
   distribution_id,
   attribution_ua,
-  COUNT(*) AS num_activated
+  COUNT(activated) AS num_activated,
```

Comment:
nit: because `COALESCE(..., FALSE)`
```suggestion
  COUNTIF(activated) AS num_activated,
```